cJSON.h

```diff
@@ -266,6 +268,7 @@ CJSON_PUBLIC(cJSON*) cJSON_AddTrueToObject(cJSON * const object, const char * co
 CJSON_PUBLIC(cJSON*) cJSON_AddFalseToObject(cJSON * const object, const char * const name);
 CJSON_PUBLIC(cJSON*) cJSON_AddBoolToObject(cJSON * const object, const char * const name, const cJSON_bool boolean);
 CJSON_PUBLIC(cJSON*) cJSON_AddNumberToObject(cJSON * const object, const char * const name, const double number);
+CJSON_PUBLIC(cJSON*) cJSON_AddNumberToObjectOpts(cJSON * const object, const char * const name, const double n, int places);
```

Comment:
I think the name is too generic, why not `cJSON_AddNumberWithPrecisionToObject`?

Reply:
cJSON_AddNumberWithPrecisionToObject is descriptive for sure, maybe its just me but name seems a long, I am happy to rename the API though.